GOVERNANCE.md

```diff
@@ -98,19 +106,35 @@ There is no special process for changing this document. Submit a PR and ask for
 
 Please also make a note in the change log under [`# Governance history`](#governance-history)
 
-# Code of conduct
+# Finances and Funding
+
+There is currently no mechanism for the data.table project to receive funding as an entity.  
+
+Funding support for this project therefore may come in two forms:
+
+## Individual external funding
 
-As contributors of this project, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+Any individual developer or community member of data.table may apply for and receive funding for their work on the project.  Individuals or groups seeking funding support are strongly encouraged to consult directly with the data.table Project Members (by initiating an Issue on GitHub) to ensure funds are used meaningfully. Formally, however, decisions about use of funds are governed by the individual grantee(s) and their contract with the funding agency. 
 
-We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, etc.
+There is no guarantee that funded work will be incorporated into the data.table package; any contributions, whether funded or unfunded, are subject to the same review process as outlined above.
 
-Examples of unacceptable behavior by participants include the use of sexual language or imagery, derogatory comments or personal attacks, trolling, public or private harassment, insults, or other unprofessional conduct.
+## Direct donations
 
-Committers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct. A person with special roles who does not follow the Code of Conduct may have their roles revoked.
+Direct donations to the project may be made via GitHub Sponsorships, which allow individuals to fund a specific developer.  Those wishing to support the project in general, rather than a specific developer, should direct their donations to the current CRAN Maintainer.
```

Comment:
If I'm reading correctly, Tyson would need to have Sponsorships activated, is that right? (He may have done so already, I couldn't tell from glancing at the profile).

Reply:
Yes, we're chatting about this tomorrow to see if he's willing/able to set that up - but first wanted to see how the team felt about this mechanism.

Reply:
I think it will be helpful for transparency to specify what are the acceptable uses of funds allocated in this way. We don't need to get it right (if we think of a smart new way to apply funding), but we can always update the doc.